pkgs/development/python-modules/fpylll/default.nix

```diff
@@ -14,13 +14,14 @@
 
 buildPythonPackage rec {
   pname = "fpylll";
-  version = "0.4.1dev";
+  version = "0.5.0dev";
 
+  # src = lib.cleanSource /home/timo/repos/fpylll;
```

Comment:
Do you want to keep it for the next time you need to patch something?

Reply:
No, I just used that to bisect the issue. Didn't mean to commit it. Good catch!